webrtc.html

```diff
@@ -6605,6 +6605,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       SSRC is always updated, and if the RTP packet contains CSRCs, then the
       <code><a>RTCRtpContributingSource</a></code> objects corresponding to
       those CSRCs are also updated.</p>
+      <p>All of the <code><a>RTCRtpContributingSource</a></code> objects that
+      correspond to the same <code><a>RTCRtpReceiver</a></code> MUST be
+      updated at the same time within the same microtask checkpoint, or updated
+      in a different but indistinguishable manner. This is necessary so that an
```

Comment:
This seems stricter than it needs to be, and there's an algorithm hiding here: Browsers must queue a task to update this information.

When that task runs, it runs to completion, including running and emptying everything on the micro-task queue, *which can only contain things put there by the algorithm itself*, before content JS runs again. So no need to dictate consistency at the microtask level.

I think we should specify that algorithm here.